src/components/LayerUI.tsx

```diff
@@ -81,6 +73,39 @@ interface LayerUIProps {
   children?: React.ReactNode;
 }
 
+export const mainMenuTunnel = tunnel();
+export const welcomeScreenMenuHintTunnel = tunnel();
+export const welcomeScreenToolbarHintTunnel = tunnel();
+export const welcomeScreenHelpHintTunnel = tunnel();
+export const welcomeScreenCenterTunnel = tunnel();
+export const footerCenterTunnel = tunnel();
```

Comment:
Its better to create the tunnels where its going to be used and export from there eg mainMenuTunnnel should to `MainMenu.tsx` instead